_pages/mobile-interview-project.md

```diff
@@ -89,63 +144,108 @@ Feel free to either:
+
+ * Q) **Does UI design / UI appearance matter?**
+
+Do not worry about building custom controls. Using standard system UI controls are totally fine.
+
+There’s no need to make the UI look amazing, but if that’s a strength of yours, definitely please show us!
 
-**Q) Can I utilize code I’ve written for my personal / side projects?**
-Do feel free to copy in (or pull in as a dependency) utility functions, classes, types (etc) you usually bring into your personal projects, as long as you wrote them.
+ * Q) **Does on-device performance matter?**
 
-Please **do not** use code from previous/current employers, for obvious reasons :-)
+Sort of: phones and tablets are pretty fast these days, so there’s no need to prematurely optimize anything or do microbenchmarks. Please do try to make sure that there aren’t any problems when using the app (like dropped frames, freezing).
 
-**Q) What programming languages can I use?**
-Please utilize Java or Kotlin for Android, or Objective-C or Swift for iOS. For iOS, if you need to drop down to CoreFoundation, Accelerate, etc, C is fine too (but you shouldn’t need to).
+That being said, do consider network usage: we will want to lazy load images in order to avoid wasting the user’s data plan, and not call the employee list service more than necessary.
 
-Please **do not** utilize cross-platform SDKs such as React Native, Flutter, Phonegap, Cordova, etc.
+ * Q) **For iOS, can I use xibs and storyboards?**
 
-**Q) Can I use 3rd party frameworks?**
-Sure – feel free to use any “no brainer” frameworks you’d usually pull into an app such as RxJava, RxSwift, etc. However, since this is a relatively simple application, you shouldn’t need many external dependencies. In particular, we’d like to see no 3rd party image caching framework used for loading and storing employee’s profile images (since this is an interesting part of the question).
+Feel free to do so. However, note that we do not (yet) write code in Swift UI or Compose at Square. If you use the Compose beta, please let us know how to build and run your app.
 
-...And of course, 1st-party frameworks that come with iOS or Android are fine.
+ * Q) **Can I use SwiftUI (iOS) or Compose (Android)?**
 
-**Q) What build systems & build tooling should I use?**
-Please utilize platform-standard build systems and tooling. If you decide to use tooling such as CocoaPods or Bazel to assist in project configuration and set up, include a README explaining how to build and run the app.
+Feel free to do so. However, note that we do not (yet) write code in Swift UI or Compose at Square. If you use the Compose beta, please let us know how to build and run your app.
```

Comment:
I think we do/are starting to write compose as of now.. or soon.

Also, I think Compose is no longer in Beta 🎉